R/adae.R

```diff
@@ -1,6 +1,5 @@
-#' Dataset adae
+#' Adverse Event Analysis
```

Comment:
please update ADAE label in [adams-specs.xlsx](https://github.com/pharmaverse/pharmaverseadam/blob/main/inst/extdata/adams-specs.xlsx)

![image](https://github.com/pharmaverse/pharmaverseadam/assets/47867131/e9f21490-8ec8-4d5a-8bea-7ac3bd76a5b6)